env/packages.yml

```diff
@@ -46,8 +46,11 @@
       when: ansible_kernel | match(".*-generic")
 
     - name: Install list of Python packages
-      pip: name=grpcio
+      pip: name={{item}}
       become: true
+      with_items:
```

Comment:
I am concerned with `with_items` (https://github.com/NetSys/bess/pull/361). Instead, How about using `apt` (`python-pep8` and `python-autopep8`)?

Note that `python-autopep8` package alone is not enough; its dependency package `pep8` would only support Python 3, so `python-pep8` should be also installed.